src/api/vade_evan_api.rs

```diff
@@ -486,6 +486,95 @@ impl VadeEvan {
             .map_err(|err| err.into())
     }
 
+    /// Revokes a given credential with the help of vade and updates revocation list credential
+    ///
+    /// # Arguments
+    ///
+    /// * `credential` - credential to be revovked as serialized JSON
+    /// * `update_key_jwk` - update key  in jwk format as serialized JSON
```

Comment:
just a space too much

```suggestion
    /// * `update_key_jwk` - update key in jwk format as serialized JSON
```